datastore/settings/settings.py

```diff
@@ -61,9 +67,11 @@
     "rest_framework",
     "django_filters",
     "drf_spectacular",
+    "debug_toolbar",
 ]
 
 MIDDLEWARE = [
+    "debug_toolbar.middleware.DebugToolbarMiddleware",
```

Comment:
Can this be in the dev settings? https://github.com/ThreeSixtyGiving/datastore/tree/master/datastore/settings

Reply:
That's a point, I thought about trying to enable/disable it based on DEBUG, but it doesn't seem reliable to use other settings (which may be set later) in a settings file.
But just putting it in dev settings sounds like a plan, i'll try it and see.

Reply:
Done